quickstart-node/README.md

```diff
@@ -50,4 +53,121 @@ If you did not set a port environment variable, the application will start runni
 
 ## Features
 
-TODO: Add features
+This app showcases how to use various Monday app functionalities, including:
+
+### Monday Code Features
+
+* Using Monday apps Storage and Secure Storage
+* Using Monday apps Secrets and Environment variables
+* Using Monday apps Queue
+* Using Monday code Logger
+
+**NOTE**: Storage feature does not work locally, you can only check it after you push your code to Monday Code
+
+### Monday API SDK Usage
+
+* Example of updating column value in an item
+* Example of a custom query with types generate
+* Example of monday api error handling
```

Comment:
grammer please :) 
An example of updating a column value in an item.
An example of a custom query with generated types.
An example of error handling with the Monday API.